doc/classes/GIProbe.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GIProbe" inherits="VisualInstance" category="Core" version="3.2">
 	<brief_description>
+		Real-time global illumination probe.
 	</brief_description>
 	<description>
+		GIProbes are used to provide high-quality real-time indirect light to scenes. They precompute the effect of objects that emit light and the effect of static geometry to simulate the behavior of complex light in real time. GIProbes need to be baked before using, however, once baked, dynamic objects will receive light from them. Further, lights can be fully dynamic or baked.
```

Comment:
I'd use `[GIProbe]` and `[GIProbe]s` consistently to have similar formatting everywhere, or alternatively "GI probes" when you want to talk about actual probes and not the Node itself.

Reply:
Second "real time" lacks a hyphen.